From 2e26b3e95ea8042448838d9dac14313b0e7e4263 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 23 Jul 2021 08:51:32 +0200
Subject: [PATCH 054/N] tests: fix test_bytes

%p has different casing with mingw-w64, but it's implementation defined.
Change the test to the mingw-w64 variant.
---
 Lib/test/test_bytes.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Lib/test/test_bytes.py b/Lib/test/test_bytes.py
index b27d436..f18d38c 100644
--- a/Lib/test/test_bytes.py
+++ b/Lib/test/test_bytes.py
@@ -1168,7 +1168,7 @@ class BytesTest(BaseBytesTest, unittest.TestCase):
 
         if os.name == 'nt':
             # Windows (MSCRT)
-            ptr_format = '0x%0{}X'.format(2 * sizeof_ptr)
+            ptr_format = '0x%0{}x'.format(2 * sizeof_ptr)
             def ptr_formatter(ptr):
                 return (ptr_format % ptr)
         else:
